libs/Microsoft.MixedReality.WebRTC.Unity/Assets/Microsoft.MixedReality.WebRTC.Unity/Scripts/Media/LocalAudioSource.cs

```diff
@@ -85,7 +86,13 @@ private void OnPeerInitialized()
         }
 
         private async void DoAutoStartActions(WebRTC.PeerConnection nativePeer)
-        {
+        {
+            // If a headset is active then do not capture local streams.
+            if (XRDevice.isPresent)

```

Comment:
I don't think this is the right check. Will research.

Reply:
What's this check for? Why prevent local audio capture from XR devices?

Reply:
There's no problem capturing local audio from the device. This check is unnecessary.

Reply:
Ok will delete, thanks!

Reply:
Deleted.